src/main/java/bio/terra/service/filedata/DrsService.java

```diff
@@ -262,16 +285,25 @@ private DRSObject drsObjectFromFSFile(
             .accessUrl(httpsAccessURL)
             .region(bucketResource.getRegion().toString());
 
-    List<DRSAccessMethod> accessMethods = new ArrayList<>();
-    accessMethods.add(gsAccessMethod);
-    accessMethods.add(httpsAccessMethod);
+    return List.of(gsAccessMethod, httpsAccessMethod);
+  }
 
-    fileObject
-        .mimeType(fsFile.getMimeType())
-        .checksums(fileService.makeChecksums(fsFile))
-        .accessMethods(accessMethods);
+  private List<DRSAccessMethod> getDrsAccessMethodsOnAzure(FSFile fsFile) {
+    DRSAccessURL accessURL = new DRSAccessURL().url(fsFile.getGspath());
 
-    return fileObject;
+    AzureStorageAccountResource storageAccountResource =
+        resourceService.lookupStorageAccountMetadata(fsFile.getBucketResourceId());
+
+    AzureRegion region = storageAccountResource.getRegion();
+    String accessId = "az-" + region.getValue();
+    DRSAccessMethod httpsAccessMethod =
+        new DRSAccessMethod()
+            .type(DRSAccessMethod.TypeEnum.HTTPS)
+            .accessUrl(accessURL)
+            .accessId(accessId)
+            .region(region.toString());
+
+    return Collections.singletonList(httpsAccessMethod);
```

Comment:
Now that we have `List/Map/Set.of()`, it's generally a better form to use than the old `singleton` methods.
```suggestion
    return List.of(httpsAccessMethod);
```

Reply:
Ah fair, ok, yeah, I'll switch to `List.of`